src/io-methods/s3.js

```diff
@@ -68,6 +81,9 @@ exports.writeManifest = function(filePath, data) {
       })
     }
   })
-
   return Promise.all([jsonPromise, jsPromise])
 }
+
+function isDigitalOcean(filePath) {
```

Comment:
possible premature optimization here (so probably disregard):

maybe turn this function into one that returns the necessary prefix (e.g. `spaces://`) so that to add additional services is just as simple as adding them to this function?

but I'm not sure it's even worth the effort since there probably aren't a ton of "s3 compatible" services out there that people would want to use as alternatives.

Reply:
yeah i'd rather wait until there's a reason for it.